docs/sysadmin-basics/workspaces.txt

```diff
@@ -0,0 +1,71 @@
+.. _workspaces:
+
```

Comment:
Changes here: https://hpe-my.sharepoint.com/:t:/p/gene_hartsell/EaLc9Ikx2nNDrPkww9Dk8uQB0IwgEAUI3JE8P5or7-kJLg?e=4XPpPL
Do a `diff` to see suggested changes